src/NuGet.Clients/NuGet.Console/WpfConsole/WpfConsole.cs

```diff
@@ -851,7 +873,7 @@ public object Content
             public async Task WriteProgressAsync(string operation, int percentComplete)
             {
                 await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
```

Comment:
This switch is now redundant. 
You only need to switch before calling sync methods, or methods that depend on STA bound objects, so lead to a cast.

Reply:
Addressed.